kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -323,10 +323,6 @@ oriented data synchronization.
         this.$emit('stopTracking', ...args);
       },
     },
-    beforeDestroy() {
-      // Make sure any unsaved data is captured before tear down.
-      this.saveAttemptLogMasterLog();
```

Comment:
Yeah, I guess the problem here is that we just want to save the data, whereas this action saves the data, and then sets it.

I agree with @indirectlylit's comments on #2355, that we should avoid use of beforeDestroy, so I am wondering if we could try to achieve this same 'data persistence cleanup' in the actions instead.